src/components/activities/manageBillActivity/ManageBillActivity.tsx

```diff
@@ -0,0 +1,288 @@
+import { Receipt, Search } from "@material-ui/icons";
+import classNames from "classnames";
+import React, { FC, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useDispatch, useSelector } from "react-redux";
+import { TUserCredentials } from "../../../state/main/types";
+import { IState } from "../../../types";
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+} from "../../accessories/accordion/Accordion";
+import Button from "../../accessories/button/Button";
+import AppHeader from "../../accessories/appHeader/AppHeader";
+import BillFilterForm from "../../accessories/billFilterForm/BillFilterForm";
+import Footer from "../../accessories/footer/Footer";
+import RouterTabs from "../../accessories/tabs/RouterTabs";
+import { TTabConfig } from "../../accessories/tabs/types";
+import "./styles.scss";
+import Add from "@material-ui/icons/Add";
+import { Redirect } from "react-router";
+import { IBillSummary, TActivityTransitionState } from "./types";
+import { BillTable } from "../../accessories/billTable/BillTable";
+import { searchBills } from "../../../state/bills/actions";
+import { computeBillSummary, initializeBillFilter } from "./config";
+import ManageBillActivityContent from "../manageBillActivityContent/ManageBillActivityContent";
+
+export const ManageBillActivity: FC = () => {
+  const { t } = useTranslation();
+
+  const breadcrumbMap = {
+    [t("nav.dashboard")]: "/",
+    [t("nav.billing")]: "/billing",
+    [t("nav.newbill")]: "/bills",
+    [t("nav.managebills")]: "/managebills",
+  };
+  const dispatch = useDispatch();
+
+  const [expanded, setExpanded] = useState<string | false>("form");
+  const [isOpen, setIsOpen] = useState(false);
+  const [fromDate, setFromDate] = useState(
+    new Date().setDate(new Date().getDate() - 6) + ""
+  );
+  const [toDate, setToDate] = useState(new Date().setHours(23) + "");
+  const [patientCode, setPatientCode] = useState(0);
```

Comment:
I think that you could hanlde this as an object inside a single useState statement, I think this could be more readable.

new Date().setHours(23) -> why setHours?

Reply:
Ok.